contracts/dao/neutron-staking-tracker/src/testing/tests.rs

```diff
@@ -216,6 +216,81 @@ fn test_after_validator_created_with_mock_query() {
     );
 }
 
+#[test]
+fn test_after_validator_removed_during_migration() {
```

Comment:
I think it makes sense to
1. bump contract's version in cargo.toml (`0.3.0` probably?)
2. leave a note that this test is only for `0.2.1 -> 0.3.0` migration

Reply:
Agree in general but:
1. I think it must be 0.2.2, since it's just a bug fix, no interface/api/features changes
2. Agree